src/group/group.service.ts

```diff
@@ -27,4 +28,43 @@ export class GroupService {
 
     return newGroup.id;
   }
+
+  async updateGroup(
+    user: User,
+    {
+      name,
+      visibility,
+      groupId,
+    }: {
+      name: string | undefined;
+      visibility: string | undefined;
```

Comment:
How about using `name?: string` instead?

Reply:
It's good idea!